deploy-storybook/action.yml

```
@@ -30,28 +30,50 @@ inputs:
     description: Command to build Storybook with
     required: false
     default: storybook:build
+  use-prebuilt-package:
+    description: Whether if a prebuilt Storybook package should be used
+    required: false
+    default: 'false'
+  use-prebuilt-image:
+    description: Whether if a prebuilt Storybook Docker image should be used
+    required: false
+    default: 'false'
+  jenkins-folder-name:
+    description: Jenkins folder where the deployment jobs are located
+    required: false
 
 runs:
   using: composite
   steps:
     - name: Install Dependencies
+      if: ${{ inputs.use-prebuilt-package == 'false' && inputs.use-prebuilt-image == 'false' }}
       uses: toptal/davinci-github-actions/yarn-install@v4.4.2
 
     - name: Build Storybook
+      if: ${{ inputs.use-prebuilt-package == 'false' && inputs.use-prebuilt-image == 'false' }}
+      run: yarn "$BUILD_COMMAND"
       shell: bash
       env:
         BUILD_COMMAND: ${{ inputs.build-command }}
-      run: yarn "$BUILD_COMMAND"
 
     - name: Specify Repository and Release Name
       id: repo
       shell: bash
       run: |
-        echo ::set-output name=repository_name::${{ github.event.repository.name }}
-        echo ::set-output name=release_name::${{ github.event.repository.name }}-pr-${{ github.event.number || github.event.issue.number }}-storybook
+        folder_name=${{ inputs.jenkins-folder-name }}
+
+        if [[ "$folder_name" == '' ]]; then
+          echo folder_name=${{ github.event.repository.name }} >> $GITHUB_OUTPUT
+        else
+          echo folder_name=${{ inputs.jenkins-folder-name }} >> $GITHUB_OUTPUT
```

and also use the env variable here 🙇 
```suggestion
          echo folder_name=$FOLDER_NAME >> $GITHUB_OUTPUT
```